Day2/Exercise1/Solution.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import "fmt"
+
+func Frequency(s string) map[rune]int {
+	m := map[rune]int{}
+	for _, r := range s {
+		m[r]++
+	}
+	return m
+}
+func countFreq(stream []string) map[rune]int {
+
+	sum := map[rune]int{}
+	count := len(stream)
+	results := make(chan map[rune]int, count)
+
+	for _, s := range stream {
+		go func(s string) {
+			results <- Frequency(s)
+		}(s)
+	}
+
+	for i := 0; i < count; i++ {
+		for r, freq := range <-results {
+			sum[r] += freq
+		}
+	}
+
+	return sum
```

Comment:
when you display this , you won't get the desired output . Hence declare the map to have keys as strings otherwise the characters will be displayed in the ASCII format . So all you need to do here is convert the individual characters which are byte/rune to type string